README.md

```diff
@@ -32,3 +32,22 @@ end
 Documentation can be generated with [ExDoc](https://github.com/elixir-lang/ex_doc)
 and published on [HexDocs](https://hexdocs.pm). Once published, the docs can
 be found at [https://hexdocs.pm/matrix_sdk](https://hexdocs.pm/matrix_sdk).
+
+## Basic Usage
```

Comment:
@fancycade This is great! would you mind making a separate PR for this? I think it should be a part of #45. I think we should also have an example with a guest account.